python/cudf/cudf/tests/test_repr.py

```diff
@@ -219,6 +219,7 @@ def test_groupby_MI(nrows, ncols):
     pd.options.display.max_rows = nrows
     pd.options.display.max_columns = ncols
     assert gdg.__repr__() == pdg.__repr__()
+    pd.options.display.max_columns = len(gdg._data.names) - 1
```

Comment:
What is this change? This looks like modifying the test to pass on broke behavior.